doc/project/release_process.rst

```diff
@@ -85,24 +86,154 @@ At that point, the whole process starts over again.
+owner to manage execution within a subsystem or by a hardware vendor/maintainer.
+
+
+Severity
+--------
+The extent to which an issue or defect can affect the software and defines the
+impact that a given defect has on the system.
+
+The following severities shall be used:
+
++----------+------------------------------------------------------------------+
+| Severity | Description                                                      |
++==========+==================================================================+
+| Critical | Bug results in a crash of the system or one or more components   |
+|          | of the system. If the failed function is unusable and there is   |
+|          | no acceptable alternative method to achieve the advertised       |
+|          | desired outcome then the severity will be stated as critical.    |
++----------+------------------------------------------------------------------+
+| Major    | As with critical, but an acceptable alternative exists.          |
++----------+------------------------------------------------------------------+
+| Moderate | Causes the system to produce incorrect, incomplete, or           |
+|          | inconsistent results.                                            |
++----------+------------------------------------------------------------------+
+| Minor    | No crashes or damage to usability of the system and desired      |
+|          | results can be easily obtained by working around the defects.    |
++----------+------------------------------------------------------------------+
+| Cosmetic | Related to look and feel of the system.                          |
++----------+------------------------------------------------------------------+
+
+Severity shall be set based on the context of the bug and not in relation to the
+overall impact on the codebase. For example, a bug in an optional subsystem that
+results in a crash shall be marked as `critical` even if the bug does not impact
+every user of Zephyr.
+
+Blocker Bugs
+============
+
+Blocker bug process kicks in during the release process and is in effect after the
+feature freeze milestone. An issue labeled as a blocker practically blocks a
+release from happening. All blocker bugs shall be resolved before a release is
+created.
 
-The current backlog of prioritized bugs shall be used as a quality metric to
-gate the final release. The following counts shall be used:
+A bug that is granted `blocker` status can be merged to 'main' and included in
+the release all the way until the hard freeze (RC3) milestone.
 
-.. csv-table:: Bug Count Release Thresholds
-   :header: "High", "Medium", "Low"
-   :widths: auto
+Bugs of moderate severity and higher are typically the candidates to be promoted
+to blocker bugs.
 
+The following guidelines shall be followed for release blocker bugs:
 
-   "0","<20","<50"
+- Only mark bugs as blockers if the software (Zephyr) must not be released with
+  the bug present.
```

Comment:
does the term "Zephyr" here include libraries?  To mean that reads more on the specific core kernel library than the whole ecosystem.  Just looking for clarification if, for example, some critical issue is found with LittleFS (or another module library) would that fall under this umbrella?